src/ray/object_manager/ownership_based_object_directory.cc

```diff
@@ -284,6 +284,9 @@ ray::Status OwnershipBasedObjectDirectory::SubscribeObjectLocations(
           location_info, object_id,
           /*location_lookup_failed*/ !location_info.ref_removed());
       if (location_info.ref_removed()) {
+        RAY_LOG(ERROR)
```

Comment:
Maybe use INFO?

Reply:
I think ERROR is appropriate here because it is a real bug (we're using ERROR in the other spot where this flag is checked).

Reply:
Hmm I see. Didn’t know it is considered a bug. Why does it happen? Are we having an issue to track it?

Reply:
Yeah, it's this issue: https://github.com/ray-project/ray/issues/17380

For borrowed object refs, we should keep the ref count as long as we might have to reconstruct it, but right now we're only keeping the ref count while there are active ref holders.